dist/src/lib/BasicAdmin.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function BasicAdmin(props: any): import("react/jsx-runtime").JSX.Element;
```

Comment:
none of these dist/src files should exist
we only export the entire library in a single file

Reply:
Grand, no worries - I'll check if they're only being created because of running the tests

Reply:
Grand, no worries - I'll check if they're only being created because of running the tests